### PR TITLE
Adjusted heading ratio to Perfect Fifth (1.5) (Issue #4965)

### PR DESCRIPTION
**Pull request by twsp**
## Description
Changed ratio of em for h1 through h6 to follow a Perfect Fifth (1.5) ratio.  This was recommended as a design change in issue #4965.

## How Has This Been Tested?
This has been tested by using a local WordPress environment to create and view all header sizes.

## Screenshots (jpeg or gifs if applicable):
2018-02-12 15_31_20-sample title _ add new post testsite wordpress

## Types of changes
This is a non-breaking design change that only affects the size of headers in the Gutenberg editor.

## Checklist:
- [X] My code is tested.
- [X] My code follows the WordPress code style.
- [X] My code has proper inline documentation.

### CODE REVIEW

**Review by ntwb — COMMENTED** [review]
...

**Review by ntwb — CHANGES_REQUESTED** [review]
@twsp It looks like some formatting issues have crept in, could you update the PR please to use tabs for indentation and not spaces and keep the existing blanks lines please 😄

**Review by ntwb — CHANGES_REQUESTED** [review]
See above comment

**Review by ntwb — APPROVED** [review]

File: blocks/library/heading/editor.scss [hunk 1]
```diff
@@ -9,26 +9,26 @@
...
 	h2 {
-		font-size: 1.6em;
+		font-size: 3.375em;
 	}
 
 	h3 {
-		font-size: 1.4em;
-	}
+		font-size: 2.25em;
+	}
```
Comment (ntwb) [inline blocks/library/heading/editor.scss]: The extra spaces above also need to be removed please

**Review by ntwb — CHANGES_REQUESTED** [review]

**Comment by jasmussen** [comment]
Thank you for creating a pull request! However the typographic scale hasn't yet been decided upon in https://github.com/WordPress/gutenberg/issues/4965, and "Perfect Fifth" feels way too big for Gutenberg. Additionally when we are ready to implement, we need to also consider a separate set of sizes for mobile, as well as include the page title as a H1. Apologize for being so late to review this, and I'd love to help you with this.

**Comment by karmatosed** [comment]
For now, lets close this as we're still undecided on direction. I do in saying this want to be clear we can always reopen any pull request or issue. Once we decide we can then have a PR for #4965

**Pull request closed without merging**